### PR TITLE
Add note to bug-report template

### PR DESCRIPTION
**Pull request by Youssef1313**
I noticed some issues being reported that were already fixed in the latest NuGet package, but hasn't yet shipped into the SDK.

Hoping this note can eliminate the number of these reports.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ assignees: ''
 
 **Version**: [SDK 5.0.100](https://dotnet.microsoft.com/download/dotnet/5.0)
 
+<!--
+Note: Updates to NuGet package are more frequent than SDK, which means the NuGet package has fixes that didn't yet made into the SDK.
```
Comment (Evangelink) [inline .github/ISSUE_TEMPLATE/bug-report.md:21]:
```suggestion
Note: Updates to NuGet package are more frequent than SDK, which means the NuGet package has fixes that haven't yet made it into the SDK.
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -17,6 +17,11 @@ assignees: ''
...
+So it's preferred you check whether the bug you encountered has been fixed in the latest NuGet package or not.
```
Comment (Evangelink) [inline .github/ISSUE_TEMPLATE/bug-report.md:22]:
```suggestion
      Please check whether you can reproduce the bug you encountered in the latest NuGet package.
```

**Review by Evangelink — APPROVED** [review]
2 small changes but LGTM

@mavasani Could you review please?

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (Evangelink) [reply .github/ISSUE_TEMPLATE/bug-report.md:21]: Indent is to have a uniform look with the other note later in the file.

**Review by mavasani — APPROVED** [review]

**Pull request merged**